pom.xml

```diff
@@ -73,41 +70,47 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>aws-java-sdk</artifactId>
-            <version>1.11.264</version>
+            <version>1.11.687</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>apache-httpcomponents-client-4-api</artifactId>
-            <version>4.5.3-2.1</version>
+            <version>4.5.10-2.0</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>copyartifact</artifactId>
-            <version>1.37</version>
+            <version>1.43</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.main</groupId>
             <artifactId>maven-plugin</artifactId>
-            <version>3.1</version>
+            <version>3.4</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-io</groupId>
+                    <artifactId>commons-io</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>structs</artifactId>
-            <version>1.2</version>
+            <version>1.20</version>
         </dependency>
     </dependencies>
 
     <repositories>
```

Comment:
You can also simply remove this, as the `https` url is defined on the parent pom.

Reply:
Where? I don't see it defined.

Reply:
https://github.com/jenkinsci/pom/blob/2c3e0968fbd6bfed327bf33c9208b99cde78ca7e/pom.xml#L73

Reply:
But that's the pom parent for non-plugin components, right?

https://github.com/jenkinsci/pom/blob/2c3e0968fbd6bfed327bf33c9208b99cde78ca7e/pom.xml#L11-L13

Reply:
sorry, wrong link, same result:

https://github.com/jenkinsci/plugin-pom/blob/182757b442a0a97ce77a276434229301a8b1982b/pom.xml#L758

Reply:
I've checked removing it and the effective-pom is different. I'd prefer to leave it as is.